frontend/packages/dev-console/src/components/topology/__tests__/topology-test-data.ts

```diff
@@ -1287,6 +1290,112 @@ export const sampleStatefulSets: Resource = {
     },
   ],
 };
+
+export const samplePipeline = {
+  data: [
+    {
+      apiVersion: 'tekton.dev/v1alpha1',
+      kind: 'Pipeline',
+      metadata: {
+        creationTimestamp: '2019-10-18T10:06:37Z',
+        generation: 1,
+        name: 'hello-world-pipeline',
+        namespace: 't-s',
+        resourceVersion: '371236',
+        selfLink: '/apis/tekton.dev/v1alpha1/namespaces/t-s/pipelines/hello-world-pipeline',
+        uid: '73d7842d-975f-44ab-99e4-727b7cf097b6',
+      },
+      spec: {
+        tasks: [
+          {
+            name: 'hello-world',
+            taskRef: {
+              name: 'hello-world',
+            },
+          },
+        ],
+      },
+    },
+  ],
+};
+
+export const samplePipelineRun = {
+  data: [
+    {
+      apiVersion: 'tekton.dev/v1alpha1',
+      kind: 'PipelineRun',
+      metadata: {
+        creationTimestamp: '2019-10-18T10:07:28Z',
+        generation: 1,
+        labels: {
+          'tekton.dev/pipeline': 'hello-world-pipeline',
+        },
+        name: 'hello-world-pipeline',
+        namespace: 't-s',
+        resourceVersion: '371822',
+        selfLink: '/apis/tekton.dev/v1alpha1/namespaces/t-s/pipelineruns/hello-world-pipeline',
+        uid: 'a049c81e-ba40-4248-ac54-a2728893afcb',
+      },
+      spec: {
+        pipelineRef: {
+          name: 'hello-world-pipeline',
+        },
+        podTemplate: {},
+        serviceAccount: '',
```

Comment:
Although this is sample, lets not use serviceAccount field anymore

Reply:
fixed in https://github.com/openshift/console/pull/3014/commits/347fce3e413bd67bd89e45056e329e010fe5f1a6